.jshintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "quotmark": "single",
+  "node": true,
+  "globals": {
+    "angular": false,
+    "document": false,
+    "jasmine": false,
+    "module": false,
+    "describe": false,
+    "beforeEach": false,
+    "afterEach": false,
+    "it": false,
+    "inject": false,
+    "expect": false,
+    "spyOn": false
+  }
+}
```

Comment:
I personally prefer put the jshint option in Gruntfile like this https://github.com/gruntjs/grunt-contrib-jshint#specifying-jshint-options-and-globals

There are too many files/folders on root